doc/release/release_notes.rst

```diff
@@ -8,6 +8,25 @@
 Release Notes
 *************
 
+.. release:: Upcoming
+    :date: 2022-10-28
```

Comment:
@nebukadhezer feel free to remove teh date, taht's something we usually set at release time ;)